packages/api/src/command/medical/patient/consolidated-webhook.ts

```diff
@@ -50,8 +51,12 @@ export const processConsolidatedDataWebhook = async ({
   const apiType = Product.medical;
   const { id: patientId, cxId } = patient;
   try {
-    const settings = await getSettingsOrFail({ id: cxId });
+    const [settings, currentPatient] = await Promise.all([
+      getSettingsOrFail({ id: cxId }),
+      getPatientOrFail({ id: patientId, cxId }),
+    ]);
 
+    console.log("CURRENT PATIENT PAYLOAD: ", currentPatient.data.cxConsolidatedRequestMetadata);
```

Comment:
I think this will result in a multi-lined log, right? Let's keep logs to one line to avoid flooding the logs @ CW, please